internal/controller/controllers/bmh_agent_controller.go

```diff
@@ -976,6 +976,12 @@ func (r *BMACReconciler) reconcileSpokeBMH(ctx context.Context, log logrus.Field
 		return reconcileComplete{}
 	}
 
+	if !funk.ContainsString([]string{models.HostStatusInstalling, models.HostStatusInstallingInProgress}, agent.Status.DebugInfo.State) {
```

Comment:
HostStatusInstalled should also be included, in case the controller somehow missed the installation (e.g., it was down or stuck)